### DISCUSSION TITLE
RSUITE | A separate button to clean InputPicker

### DISCUSSION THREAD

**Discussion by danishkaleem095** [rsuite/rsuite#2385] [General]
I have multiple InputPickers in a component, need to clean all InputPickers when a separate Clear Filter button is clicked. I am not able to find any clear function of InputPicker.

**Comment by SevenOutman** [comment]
You could pass a controlled `value` prop to `<InputPicker>` and set the value to empty string when clicking your button.

Demo: https://codesandbox.io/s/clear-inputpicker-ukomin
